source/TimeSeries/IntegrationTests/Fixtures/TimeSeriesFunctionAppFixture.cs

```diff
@@ -31,14 +31,20 @@ namespace Energinet.DataHub.TimeSeries.IntegrationTests.Fixtures
 {
     public class TimeSeriesFunctionAppFixture : FunctionAppFixture
     {
+        private const string TimeSeriesData = "timeseries-data";
+
+        private const string TimeSeriesRaw = "timeseries-raw";
```

Comment:
nit
```suggestion
        private const string TimeSeriesDataFolderName = "timeseries-data";

        private const string TimeSeriesRawFolderName = "timeseries-raw";
```
Or what is the proper term if not "folder"?

Reply:
timeserie-data is a container
timeserie-raw we can call folder

I will change the naming